### PR TITLE
rds-improve

### PR DESCRIPTION
**Pull request by Joeper214**
### Goal

- [x] Use rds client version 2.
- [x] Shorten rds details 
- [x] new UI for RDS tabpane
- [x] RDS JSON data
- [x] display rds security groups
- [x] change rds security groups
- [x] iT18N

### CODE REVIEW

**Comment by Joeper214** [comment]
Test is broken. :(

**Comment by Joeper214** [comment]
I don't know how to fix test. Please Help @pocke . And review PR. :bow:

**Comment by Joeper214** [comment]
Test is very difficult to fix. :confounded: ..

**Comment by pocke** [comment]
Can you run RSpec with your local PC ?

File: app/controllers/infrastructures_controller.rb [hunk 1]
```diff
@@ -189,15 +189,14 @@ def delete_stack
   def show_rds
     physical_id = params.require(:physical_id)
     infra_id    = params.require(:id)
+    sc_g = infra.ec2.describe_security_groups().to_h
+    security_groups = map_security_groups(sc_g, elb.security_groups)
```
Comment (pocke) [inline app/controllers/infrastructures_controller.rb]: Why do you use `elb.security_groups`? I think this should be a rds.

File: app/controllers/infrastructures_controller.rb [hunk 2]
```diff
@@ -186,34 +186,28 @@ def delete_stack
     render text: I18n.t('infrastructures.msg.delete_stack_started'), status: 202 and return
   end
 
+  # GET /infrastructures/:id/show_rds
   def show_rds
     physical_id = params.require(:physical_id)
-    infra_id    = params.require(:id)
+    sc_g = @infrastructure.ec2.describe_security_groups.to_h
```
Comment (skyarch-networks) [inline app/controllers/infrastructures_controller.rb]: __[rubocop]__
* Lint/UselessAssignment: Useless assignment to variable - `sc_g`. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)

**Comment by Joeper214** [comment]
`bundle exec rspec` is the command right?

**Comment by pocke** [comment]
Yes. 
So, we can specify file name(e.g. `bundle exec rspec spec/models/client_spec.rb`).

**Comment by pocke** [comment]
And we use the guard. 
https://github.com/guard/guard

**Comment by Joeper214** [comment]
Yes, I can run. but I haven't been using guard. Okay, I'll try guard now.

**Comment by Joeper214** [comment]
That's why running rspec locally gives different result than the CI.

File: app/models/rds.rb [hunk 3]
```diff
@@ -22,33 +22,26 @@ def initialize(infra, physical_id)
     secret_access_key = infra.secret_access_key
     region            = infra.region
 
-    @rds = ::AWS::RDS.new(
+    @rds = Aws::RDS::Client.new(
```
Comment (pocke) [inline app/models/rds.rb]: :+1:

Comment (Joeper214) [inline app/models/rds.rb]: :ocean:

File: app/views/vue/_rds_tabpane.html.erb [hunk 4]
```diff
@@ -12,27 +12,46 @@
...
       <td>Endpoint Address</td>
-      <td colspan="2">{{rds.endpoint_address}}</td>
+      <td colspan="2">{{rds.endpoint.address}}</td>
     </tr>
     <tr>
       <td>Multi AZ</td>
-      <td colspan="2">{{rds.multi_az}}</td>
+      <td colspan="2">
+        <span v-if="rds.multi_az">YES</span>
+        <span v-else="rds.multi_az">NO</span>
```
Comment (pocke) [inline app/views/vue/_rds_tabpane.html.erb]: :+1: :100:

File: app/models/rds.rb (hunk 3 above)
Comment (pocke) [inline app/models/rds.rb]: :whale2:

File: app/views/vue/_rds_tabpane.html.erb (hunk 4 above)
Comment (Joeper214) [inline app/views/vue/_rds_tabpane.html.erb]: :racehorse:

File: app/views/vue/_rds_tabpane.html.erb [hunk 5]
```diff
@@ -12,27 +12,46 @@
...
+      </td>
     </tr>
     <tr>
       <td>Database Engine</td>
       <td colspan="2">{{rds.engine}}</td>
     </tr>
+    <tr>
+      <td> <%=t('security_groups.title')%> </td>
+      <td>
+        <button href="#view-rules-modal" @click="view_rules()" class="btn btn-xs btn-info" class="btn btn-xs dropdown-toggle" data-toggle="modal">
```
Comment (pocke) [inline app/views/vue/_rds_tabpane.html.erb]: It the `href` needed?

Comment (Joeper214) [inline app/views/vue/_rds_tabpane.html.erb]: Yes. Because It triggers the modal.

Comment (pocke) [inline app/views/vue/_rds_tabpane.html.erb]: I can't find modal with the id.

Comment (Joeper214) [inline app/views/vue/_rds_tabpane.html.erb]: ごめなさい。I thought it's the modal. That `href` is not needed.

**Comment by pocke** [comment]
Back button doesn't work...
out

**Comment by Joeper214** [comment]
Okay. I'll fix it.

**Comment by pocke** [comment]
Thanks :bow:

**Comment by pocke** [comment]
merge ok ?

**Comment by Joeper214** [comment]
おねがいします。

**Comment by pocke** [comment]
マージします！

**Pull request merged**